src/reference/asciidoc/logging.adoc

```diff
@@ -270,6 +270,42 @@ public class MyEnhancedAppender extends AmqpAppender {
 ----
 ====
 
+Starting with <BACKLOG VERSION>, the log4j2 `AmqpAppender` can be extended using `@PluginBuilderFactory` and extending also
```

Comment:
There is no such a `<BACKLOG VERSION>` placeholder. Please, consider to use an explicit current version.
You can find it in the `gradle.properties`.

Also, please, follow the rule *One sentence per line*: https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line

Reply:
already fixed

Reply:
It will be fixed